test/e2e/test-apps/other/browser-tracing/package.json

```diff
@@ -4,6 +4,6 @@
   "main": "src/main.js",
   "dependencies": {
     "@sentry/electron": "3.0.0",
-    "@sentry/tracing": "7.11.1"
+    "@sentry/tracing": "7.12.1"
```

Comment:
Is there some way we can automate this? or make the e2e tests grab the local npm package instead?